server/tmserver/world.py

```diff
@@ -580,8 +576,14 @@ def handle_move(cls, sender_obj, action_args):
         # shortname of a room in the database. In the future we might need
         # fuzzy matching but for now I think moves are largely programmatic?
         room = GameObject.get_or_none(GameObject.shortname==action_args)
-        cls.put_into(room, sender_obj)
-        cls.user_hears(sender_obj, sender_obj, 'You materialize in a new place!')
+        if room:
+            if sender_obj == room:
+                cls.user_hears(sender_obj, sender_obj, "You can't move to yourself.")
```

Comment:
would we rather let the ClientException bubble up and format an `Error: ...` style message for the user?

Reply:
Not sure, feels like passing errors should be for issues that need to be fixed, where as this would be expected user shenanigans?

Reply:
word. i've been using it for the latter mainly so it's clear to the user that they did something weird. but it's also used for Actually Broken things.

cool, moving forward let's save `ClientException` for actually broken things and then maybe just color "error" output from shenanigans a little differently.